parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -18,6 +18,8 @@ done
 sed -i "/#HELPERSEOF/d" $script_lib
 source $script_lib
 
+apt_fix_keys
```

Comment:
we need to make sure this is intentionally best-effort to allow for no outbound scenarios

Reply:
I sent it to the background, which should be 100% sane, as other apt operations will wait for locks that this func holds, and retry if they run during key remediation.

In terms of no outbound scenarios, should we simply skip this operation for those scenarios? Does CSE know when we're in a no outbound context?

Reply:
For AKS Engine, yes (via the no outbound feature flag, which we can pass into cse if not already doing so). For AKS, I don't think this is part of the limited egress required addresses  https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic#required-ports-and-addresses-for-aks-clusters